src/server/ssmConfig.ts

```diff
@@ -1,31 +1,36 @@
-import { ssm } from './aws';
-import { App, Stack, Stage } from './appIdentity';
-import { Option, some, none, map, withDefault } from '@guardian/types/option';
-import { pipe2 } from 'lib';
+import type { Option } from "@guardian/types/option";
```

Comment:
I assume this makes the Option type available throughout the code base without exporting? Is this like a d.ts file?

Reply:
This is a recommended new feature in TypeScript https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html